UselessMine.cpp

```diff
@@ -220,8 +221,23 @@ void UselessMine::Init (const char* commandLine)
     bztk_registerCustomDoubleBZDB("_mineSafetyTime", 5.0);
 
     // Save the location of the file so we can reload after
-    deathMessagesFile = commandLine;
-    defusalMessagesFile = commandLine;
+    
+    // This expects two command line parameters: the death messages
+    // file and the defusal messages file. 
+    bz_APIStringList cmdLineParams;
+    cmdLineParams.tokenize(commandLine, ",");
+    
+    if (cmdLineParams.size() == 2)
+    {
+        deathMessagesFile = cmdLineParams.get(0);
+        defusalMessagesFile = cmdLineParams.get(1);
+    }
+    else
+    {
+        deathMessagesFile = "";
+        defusalMessagesFile = "";
+        bz_debugMessagef(DEBUG_VERBOSITY, "DEBUG :: Useless Mine :: Could not load messages.");
```

Comment:
Say something like "no messages loaded" otherwise it sounds like there was an error while loading the messages